lms/djangoapps/instructor_task/subtasks.py

```diff
@@ -494,7 +494,6 @@ def update_subtask_status(entry_id, current_task_id, new_subtask_status, retry_c
         _release_subtask_lock(current_task_id)
 
 
-@transaction.commit_manually
```

Comment:
BTW, this change from the deprecated `commit_manually` to `atomic` will fix a few bulk email `False is not true` failures.